docarray/utils/_internal/pydantic.py

```diff
@@ -0,0 +1,12 @@
+import pydantic
+
+is_pydantic_v2 = pydantic.__version__.startswith('2.')
+
+
+if not is_pydantic_v2:
+    from pydantic.validators import bytes_validator
+
+else:
+    from pydantic.v1.validators import bytes_validator
```

Comment:
Am I understanding correctly that we are falling back to the bytes validator of v1 even if pydantic v2 is used? What is the reason behind this?

Reply:
?